doc/nrf/libraries/zigbee/lib_zigbee_app_helper_functions.rst

```diff
@@ -0,0 +1,322 @@
+﻿.. _lib_zigbee_app_helper_functions:
+
+Zigbee application helper functions
+###################################
+
+|NCS| provides several helper functions that are intended to facilitate the end user application implementation when using the :ref:`nrfxlib:zboss` library.
+They implement the default behavior of a Zigbee device and are used in :ref:`zigbee_samples` in this SDK.
+
+.. _zarco_signal_handler:
+
+Zigbee default signal handler
+*****************************
+
+The Zigbee default signal handler is a function called from the :cpp:func:`zboss_signal_handler()` function that is mandatory for each application.
```

Comment:
I'd rephrase a bit.

```
ZBOSS stack interacts with the user application by invoking `zboss_signal_handler()` function 
whenever a stack event, such as network steering, occurs. It is mandator to define 
`zboss_signal_handler()` in the application. Because [...] signals. This function can be 
called in application's `zboss_signal_handler()` simplifying implementation.
```